LastWork(newWork)/css/p_newevaluation.css

```diff
@@ -0,0 +1,155 @@
+.image{
+	width: 150px;
+	height: 30px;
+	margin-left: 50px;
+}
+
+.header{
+	width: 100%;
+	height: 55px;
+	border-bottom: 1px solid;
+	display: inline-flex;
+	align-items: center;
+	box-shadow: 0px 3px 10px grey;
+}
+
+.header_div{
+	display: flex;
+	width: 1200px;
+	margin: 0 auto;
+}
+
+.body_section{
+	display: flex;
+	width: 1200px;
+	height: 100%;
+	margin: 0 auto;
+}
+
+.eval_button{
+	text-decoration: none;
+	padding: 0px 50px;
+	float:left;
+	margin-bottom: 0px;
+}
+
+.neweval_button{
+	border-bottom: 10px solid;
+	border-bottom-color: #03a8f3;
+	text-decoration: none;
+	padding: 0px 50px;
+	float: left;
+	margin-bottom: 0px;
+}
+
+.logout{
+	float: right;
+	text-decoration: none;
+	margin-right: 50px;
+}
+
+.footer_section{
```

Comment:
I feel you're duplicating stuff a lot. Create a single footer.css file and put the footer styles once. Also break down the selectors into more granular one, according to skin/structure separation so that you can increase re-usability